samcli/commands/sync/command.py

```diff
@@ -122,15 +115,17 @@
 @click.option(
     "--resource",
     multiple=True,
-    type=click.Choice(SyncCodeResources.values(), case_sensitive=True),
+    cls=ReplaceHelpSummaryOption,
+    replace_help_option="--resource RESOURCE",
     help=f"Sync code for all resources of the given resource type. Accepted values are {SyncCodeResources.values()}",
 )
 @click.option(
     "--dependency-layer/--no-dependency-layer",
     default=True,
     is_flag=True,
-    help="This option separates the dependencies of individual function into another layer, for speeding up the sync."
-    "process",
+    # cls=ReplaceHelpSummaryOption,
```

Comment:
What's the reason for this being commented out?

Reply:
Left behind from a refactor, next commit removes it.